src/orchestration/polling_message_handler.go

```diff
@@ -2,19 +2,28 @@ package orchestration
 
 import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azqueue"
+	"github.com/CDCgov/reportstream-sftp-ingestion/secrets"
 	"github.com/CDCgov/reportstream-sftp-ingestion/sftp"
 	"github.com/CDCgov/reportstream-sftp-ingestion/utils"
 	"log/slog"
 )
 
+// TODO - Are we adding test coverage?
+// TODO exclude from unit testing and add to integration
```

Comment:
Probably good to either clean up these test coverage TODOs or do something with them - like if we exclude something, should maybe include an explanation of why

Reply:
Removed todos added during this PR!